rewrite-maven/src/main/java/org/openrewrite/maven/ChangePluginGroupIdAndArtifactId.java

```diff
@@ -31,8 +32,6 @@
 @Value
 @EqualsAndHashCode(callSuper = false)
 public class ChangePluginGroupIdAndArtifactId extends Recipe {
-    @EqualsAndHashCode.Exclude
```

Comment:
Somewhat unrelated to the contents of this PR: I think we should add a note to the description of this recipe that no validation is done, it is assumed the values passed are valid. Even the description of newVersion property is invalid as it doesn't resolvenode or semver style selectors

Reply:
- https://github.com/openrewrite/rewrite/pull/5257